script.js

```diff
@@ -98,19 +134,175 @@ var getDefaultWinLossMessage = function () {
   );
 };
 
-// Check whether player draws with computer
+/**
+ * Check whether player draws with computer objects
+ * @param {string} playerObject
+ * @param {string} computerObject
+ */
 var doesPlayerDrawWComputer = function (playerObject, computerObject) {
+  // determine draw conditions for 'reversed sps'. Return a boolean
+  if (mode == REVERSED_SPS) {
+    return (
+      (playerObject == REVERSED_SCISSORS && computerObject == SCISSORS) ||
+      (playerObject == REVERSED_PAPER && computerObject == PAPER) ||
+      (playerObject == REVERSED_STONE && computerObject == STONE)
+    );
+  }
+  // determine draw conditions for regular sps. Return a boolean
+  return playerObject == computerObject;
+};
+
+/**
+ * Check whether player's inputs are valid
+ * @param {string} userInput
```

Comment:
The last line in JSDoc function-level comments is typically the return value.

e.g.
```
/**
 * Check whether player's inputs are valid
 * @param {string} userInput
 * @returns `true` if user input is scissors, paper, or stone, `false` otherwise
 */

Reply:
@jeremylim91 could we apply this standard to all function-level comments in our ref solns? Thank you!